pages/start.tsx

```diff
@@ -153,16 +154,28 @@ const LoginStep2: React.FC<LoginStep2Props> = ({ handleLoginStep, setRepoList })
         });
 
         setRepoList(repoList);
+        setAddOrgErrorCode(0);
       }
 
       // If valid, go to next step
       handleLoginStep();
-    } catch (e) {
+    } catch (e: any) {
       // If invalid, display error
-      console.error(e);
+      setAddOrgErrorCode(e.status);
     }
   };
 
+  const getOrgLookupError = (errorCode: number) => {
+    switch (errorCode) {
+    case 0: 
+      return <></>;
+    case 404:
+      return <Text className="bg-light-red-4 p-1">The organization is invalid</Text>;
```

Comment:
```suggestion
      return <Text className="bg-light-red-4 p-1">That organization was not found.</Text>;
```

Not found is a better explanation than "invalid"